tests/libunwind/host/host.cpp

```diff
@@ -6,11 +6,19 @@
 #include <openenclave/internal/error.h>
 #include <openenclave/internal/tests.h>
 #include <sys/types.h>
-#include <unistd.h>
 #include <cassert>
 #include <cstdio>
 #include <cstdlib>
 #include <cstring>
+
+#if defined(__linux__)
+#include <unistd.h>
+#elif defined(_WIN32)
+#include <process.h>
+#else
+#error "Unsupported OS platform"
```

Comment:
Should we put this somewhere else more global, maybe in some common header file? I don't think we support any platform but Linux & Windows right now.

Reply:
The included header is specifically required by this .cpp file, so I think this should be fine. However, I agree on we don't need the if-elif-else check for now.